Desktop_Interface/i2cdecoder.cpp

```diff
@@ -13,7 +13,15 @@ i2cDecoder::i2cDecoder(isoBuffer* sda_in, isoBuffer* scl_in, QPlainTextEdit* con
     updateTimer = new QTimer();
     updateTimer->setTimerType(Qt::PreciseTimer);
     updateTimer->start(CONSOLE_UPDATE_TIMER_PERIOD);
-    connect(updateTimer, SIGNAL(timeout()), this, SLOT(updateConsole()));
+    connect(updateTimer, &QTimer::timeout,
+            this, &i2cDecoder::updateConsole);
```

Comment:
It's not necessarily wrong (at all) to split calls like these along two lines, but I personally don't do it.  Are you able to do single line function definitions and calls just for consistency?

Reply:
Ah, my bad. I've been looking throught the Qt docs recently and that's the style they use, I guess it rubbed off onto me.

I'll put it back into a single line.